apps/explorer/src/utils/sentry.ts

```diff
@@ -11,11 +11,14 @@ import {
     useNavigationType,
 } from 'react-router-dom';
 
-import { featuresPromise, growthbook } from './growthbook';
+// const SENTRY_ENABLED = import.meta.env.PROD;
+const SENTRY_ENABLED = true;
```

Comment:
did you mean to leave this default enabled?

Reply:
Nope! 😅